### PR TITLE
Added content import

### PR DESCRIPTION
**Pull request by stephenwf**
Rebased version of #3

### CODE REVIEW

**Comment by stephenwf** [comment]
@giorgiosironi @elrossco22 This should be ready to merge now

File: salt/search/config/etc-init-search-queue-watch.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description "search-queue-watch Will listen to the SQS queue and pass items to gearman to import"
+respawn
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
+instance $ID
+chdir /srv/search
+script
+    exec php bin/console queue:watch search--{{ pillar.elife.env }} -e {{ pillar.elife.env }}
```
Comment (giorgiosironi) [inline salt/search/config/etc-init-search-queue-watch.conf]: this is missing `$ID` at the end like at https://github.com/stephenwf/search-formula/blob/93d68a7294fa2bc59c21b95ecf419b9da056edb9/salt/search/config/etc-init-search-gearman-worker.conf#L8

**Comment by giorgiosironi** [comment]
You also need to add to `processes.sls`, which is only used in `end2end` and (in the future) `prod`:
https://github.com/stephenwf/search-formula/blob/93d68a7294fa2bc59c21b95ecf419b9da056edb9/salt/search/processes.sls

Those are the Upstart tasks (not permanent services) executed on deploy that go through the services and restart them. Borrow from https://github.com/elifesciences/elife-bot-formula/blob/master/salt/elife-bot/processes.sls#L1 so that the code is not duplicated but you iterate over the map `{'gearman-worker': 3, 'queue-watch': 1}`

**Comment by stephenwf** [comment]
@giorgiosironi Okay added that in, is it worth creating a map in the `init.sls` file too?

**Comment by giorgiosironi** [comment]
Yes but there it would only be a list, since you don't need to know the number of processes:
https://github.com/elifesciences/elife-bot-formula/blob/master/salt/elife-bot/init.sls#L232

**Comment by stephenwf** [comment]
@giorgiosironi Hows that looking now?

**Comment by giorgiosironi** [comment]
:+1: for merging

**Pull request merged**